api/turing/service/ensembling_job_service.go

```diff
@@ -181,11 +255,21 @@ func (s *ensemblingJobService) CreateEnsemblingJob(
 	job.InfraConfig.ArtifactURI = ensembler.ArtifactURI
 	job.InfraConfig.EnsemblerName = ensembler.Name
 
+	project, err := s.mlpService.GetProject(projectID)
+	if err != nil {
+		return nil, err
+	}
+
+	url, err := s.generateMonitoringURL(job, project)
+	if err != nil {
+		return nil, err
+	}
+	job.MonitoringURL = url
```

Comment:
`MonitoringURL` is not persisted in the database, right? Is the value remain set after `Save(job)` is called?

Reply:
It's not persisted but let's just not take the chance that the value might be wiped, so I'll put it after persisting.